src/styles/blocks/index.scss

```diff
@@ -0,0 +1,59 @@
+@import '../utils/variables';
+
+body {
+  margin: 0;
+  padding-top: 25px;
+}
+
+.calendar {
+  display: flex;
+  gap: $gap;
+  flex-wrap: wrap;
+  width: $col-count * $cell-width + $gap * $col-count;
+  margin: 0 auto;
+
+  &__day {
+    display: flex;
+    font-family: Arial, Helvetica, sans-serif;
+    justify-content: center;
+    align-items: center;
+    font-size: 30px;
+    line-height: 100%;
+    box-sizing: border-box;
+    background-color: #eee;
+    width: $cell-width;
+    height: $cell-height;
+    border: 1px solid black;
+
+    &:first-child {
+      margin-left: ($col-count - 1) * $cell-width + $gap * ($col-count - 1);
```

Comment:
The calculation for `margin-left` assumes that `$col-count`, `$cell-width`, and `$gap` are defined and have valid values. Verify these variables are correctly set in the imported file.